run_unittests.py

```diff
@@ -350,7 +334,11 @@ def setUp(self):
         self.mconf_command = [sys.executable, os.path.join(src_root, 'mesonconf.py')]
         self.mintro_command = [sys.executable, os.path.join(src_root, 'mesonintrospect.py')]
         self.mtest_command = [sys.executable, os.path.join(src_root, 'mesontest.py'), '-C', self.builddir]
-        self.ninja_command = [detect_ninja(), '-C', self.builddir]
+        # Backend-specific commands
+        self.backend = os.environ.get('MESON_BACKEND', 'ninja')
+        self.build_command, self.clean_command, self.test_command, self.install_command, \
+                self.uninstall_command = get_backend_commands(self.backend)
```

Comment:
__[flake8]__
* [E127] continuation line over-indented for visual indent

<!-- Comment by [SideCI](https://sideci.com) -->